js/controller.js

```diff
@@ -1209,14 +1217,21 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       this.mb.publish(OO.EVENTS.SET_CURRENT_AUDIO, this.state.currentVideoId, currentTrack);
     },
 
-    onSeeked: function() {
+    onSeeked: function(event, time) {
       this.state.seeking = false;
       if (this.state.queuedPlayheadUpdate) {
         OO.log('popping queued update');
         this.skin.updatePlayhead.apply(this.skin, this.state.queuedPlayheadUpdate).then(() => {
           this.state.queuedPlayheadUpdate = null;
         }).catch(() => {OO.log('onSeeked: Could not set new state for skin')});
       }
+      if (this.state.cast.isReceiver) {
+        this.skin.updatePlayhead(time);
+        if (this.state.playerState === CONSTANTS.STATE.PLAYING) {
+          this.showControlBar();
+          this.startHideControlBarTimer();
```

Comment:
You already handle this at the `onPlaying` method, I think this should be handled by the screen that is shown at the time (playingScreen, pauseScreen, endScreen)